app/src/main/java/com/codesoom/assignment/Toy/application/ToyService.java

```diff
@@ -0,0 +1,96 @@
+package com.codesoom.assignment.Toy.application;
+
+
+import com.codesoom.assignment.Task.TaskNotFoundException;
+import com.codesoom.assignment.Toy.ToyNotFoundException;
+import com.codesoom.assignment.Toy.domain.Toy;
+import com.codesoom.assignment.Toy.domain.ToyJpaRepository;
+import org.springframework.stereotype.Service;
+
+import javax.transaction.Transactional;
+import java.util.List;
+
+/**
+ * Toy Entity의 REST API
+ * GET - getAllToy(), getToyById()
+ * POST - createToy()
+ * UPDATE - updateToy()
+ * DELETE - deleteToy()
+ */
+@Service
+@Transactional
+public class ToyService {
+
+    private final ToyJpaRepository toyJpaRepository;
+
+    /**
+     * 생성자에 ToyJpaRepository 주입
+     * @param toyJpaRepository CrudRepository 상속받은 인터페이스
+     */
+    public ToyService(ToyJpaRepository toyJpaRepository) {
+        this.toyJpaRepository = toyJpaRepository;
+    }
+
+    /**
+     * 모든 Toy 목록 가져옴
+     * @return 모든 Toy 를 담고있는 list 반환
+     */
+    public List<Toy> getAllToy(){
+        return toyJpaRepository.findAll();
+    }
+
+    /**
+     * 개별 toy 반환
+     * @param id 찾고자 하는 toy id
+     * @return
+     */
```

Comment:
리턴값 위주로 이렇게 설명해 봅시다. 메소드 JavaDoc 의 설명은 꼭 완전한 문장으로 마침표까지 써서 완성해 줍니다.
```suggestion
    /**
     * 주어진 id를 가진 장난감을 찾아 리턴합니다.
     * @param id 찾고자 하는 장난감의 id
     * @return 찾은 장난감
     */
```